lib/internal/Magento/Framework/View/Result/Layout.php

```diff
@@ -7,6 +7,7 @@
 namespace Magento\Framework\View\Result;
 
 use Magento\Framework;
+use Magento\Framework\App\Response\HttpInterface as HttpResponseInterface;
```

Comment:
Th coupling already was there before the change. The change simply made it visible.